### PR TITLE
CASSANDRA-15940 REDO: make changes to index.html to update companies, add third-party page

### PR DESCRIPTION
**Pull request by polandll**
I could not fix the repo for the previous PR, so I redid the work and reforked. Please approve this PR.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -22,20 +22,32 @@ <h3>What is Cassandra?</h3>
         <h4>Proven</h4>
         <p>
         Cassandra is in use at
+        <a href="https://www.dataversity.net/case-study-cassandra-meets-call-of-duty/">Activision</a>,
         <a href="http://www.slideshare.net/daveconnors/cassandra-puppet-scaling-data-at-15-per-month">Constant Contact</a>,
-        <a href="http://planetcassandra.org/blog/post/cassandra-at-cern-large-hadron-collider/">CERN</a>,
+
+        <!-- <a href="http://planetcassandra.org/blog/post/cassandra-at-cern-large-hadron-collider/">CERN</a>, -->
+	<!-- SWITCHED TO SCYLLA https://www.scylladb.com/2018/09/05/cern-real-time-processing-big-data-scylla/ -->
```
Comment (michaelsembwever) [inline src/index.html:29]: could we remove the `<!-- … -->` lines? there's no need to keep them, we have them in version control (git).

**Comment by michaelsembwever** [comment]
with the exception of removing the commented lines in the html, this looks good to go from me!

**Review by michaelsembwever — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (polandll) [reply src/index.html]: So sorry, had removed and committed, then had to redo.

**Comment by michaelsembwever** [comment]
merged manually with 7d367b75032f49712ec5c9c3afa178c82e8e3fb2

**Pull request closed without merging**